README.md

```diff
@@ -1,57 +1,250 @@
-- To Set up the environment and tools needed to develop a React application.
-- Configure with special linters. 
+# 📗 Table of Contents
 
-# Built With
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-- React
-- Webpack and Babel
+<!-- PROJECT DESCRIPTION -->
 
-# Key Features
+# 📖 [Math Magicians] <a name="about-project"></a>
 
-- Make simple calculations.
-- Read a random math-related quote.
+> Describe your project in 1 or 2 sentences.
 
-# Future Features
+**Math Magician** is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
 
-- To add 3 pages to the website which will display different information
+Make simple calculations.
+Read a random math-related quote.
 
-# Getting Started
+## 🛠 Built With <a name="built-with">HTML, CSS, JS</a>
 
-**To get a local copy up and running follow these simple steps.**
+### Tech Stack <a name="tech-stack">React</a>
 
-Run the command below from the terminal
+> Describe the tech stack and include only the relevant sections that apply to your project.
 
-- git clone (https://github.com/teke85/Math-Magician.git) in your terminal
-- cd Math-Magician
-- run: npm start
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
-## Demo
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
 
-Not Available
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-## Author
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
```

Comment:
- [ ] [OPTIONAL] Please consider editing your README and describing how to set up and run this project locally